charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -361,14 +361,14 @@ istio:
 # Certificate value section            #
 # -------------------------------------#
 certificate:
-      # certificate.useCustomIssuer -- If set to `true` the default issuer's configuration settings will be used. This requires a custom issuer to be installed.
-      useCustomIssuer: true
-      # certificate.customIssuerSel -- custom issuer selector
+      # certificate.customIssuer -- predefined custom issuer selector; allowed values: internet, intranet
+      customIssuer: intranet
+      # certificate.customIssuerSel -- custom issuer selector; ignored if `customIssuer` is set
       customIssuerSel:
         # certificate.customIssuerSel.name -- name of the custom issuer to be used
-        name: wadtfy-certificate-issuer
+        name:
         # certificate.customIssuerSel.kind -- kind of the custom issuer to be used
-        kind: Issuer
```

Comment:
this property would then be obsolete since the kind can only be of type Issuer when none of the default ClusterIssuers have been chosen before.